lib/openstax/salesforce/remote/os_ancillary.rb

```diff
@@ -5,10 +5,11 @@ class OsAncillary < ActiveForce::SObject
     field :product,                   from: "Product__c", as: :picklist
     field :account_type,              from: "Account_Type__c"
     field :book_name,                 from: "Book_Name__c"
-    field :course_start_date,         from: "Class_Start_Date__c", as: :date
+    field :course_start_date,         from: "Course_Start_Date__c", as: :date
```

Comment:
`Class_Start_Date__c` was deprecated in favor of `Course_Start_Date__c`, so we just switched existing field.